CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing to Chipy.org
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to the Chicago Python User Group website, which are hosted in the [Chicago Python User Group Organization](https://github.com/chicagopython). These are guidelines, not rules. Use your best judgement, and feel free to propose changes to this document in a pull request (PR).
```

Comment:
line 5, change `are` to `is`

Reply:
I'll go ahead and disagree with this one.  :wink: The word set is singular, therefore the word "is" is appropriate.  I agree with the the rest of your comments @sophm256.  Thanks for doing a more through review than me.

Reply:
[reference](https://www.writewithjean.com/2012/11/18/is-set-singular-or-plural/#:~:text=In%20the%20US%2C%20collective%20nouns%20like%20set%20are,haven%E2%80%99t%20decided%E2%80%A6.%E2%80%9D%20The%20committee%20is%E2%80%A6%20The%20jury%20disagree.)

Reply:
@elmq0022 You and are in agreement that the word "is" should be with "set". That's why I asked @nickymarino to change `are` to `is` in line 5. Specifically the phrase `which are hosted` needs to be changed to `which is hosted`. :wink:

Reply:
Looks like I read that wrong.  I didn't realize line 5 wrapped around that far.  :sweat_smile: